mon-pix/tests/unit/components/routes/login-form-test.js

```diff
@@ -93,16 +96,30 @@ describe('Unit | Component | routes/login-form', function() {
         sinon.assert.calledWith(component.addGarAuthenticationMethodToUser, externalUserToken);
       });
 
-      it('should notify updateUserError when update user authentication method fails', async () => {
-        // given
-        component.addGarAuthenticationMethodToUser.rejects(new Error());
+      context('when update user authentication method fails', () => {
 
-        // when
-        await component.authenticate();
+        it('should notify updateUserError', async () => {
+          // given
+          component.addGarAuthenticationMethodToUser.rejects(new Error());
 
-        // then
-        expect(component.isErrorMessagePresent).to.be.false;
-        expect(component.hasUpdateUserError).to.be.true;
+          // when
+          await component.authenticate();
+
+          // then
+          expect(component.isErrorMessagePresent).to.be.false;
+          expect(component.hasUpdateUserError).to.be.true;
+        });
+
+        it('should invalidate the session', async () => {
```

Comment:
Je propose du fonctionnel
```suggestion
        it('should disconnect the user', async () => {
```

Reply:
Invalider la session et déconnecter sont des synonymes. Je ne sais pas si on doit expliquer l'intention dans le test unitaire (pourquoi on fait ça) mais je veux bien qu'on en discute en point tech.

Reply:
On en a déjà discuté et mis dans les standards que les 2 approches sont acceptées
https://1024pix.atlassian.net/wiki/spaces/DEV/pages/1534361658/Standard+quipe#Titre

Si tu préfères, je suis OK avec la garder